src/main/java/seedu/address/model/attempt/Attempt.java

```diff
@@ -1,18 +1,51 @@
 package seedu.address.model.attempt;
 
+import seedu.address.model.attempt.exceptions.AttemptHasBeenAttemptedException;
 import seedu.address.model.exercise.Exercise;
+import seedu.address.model.exercise.Type;
```

Comment:
It would be better if we could call it something other than Type -- too generic

Reply:
have replaced it with `Lift`